src/Twig/PossibleTwigMethodCallsProvider.php

```diff
@@ -50,10 +48,15 @@ public function provide(): array
             Assert::directory($absoluteTwigTemplatePath);
             Assert::fileExists($absoluteTwigTemplatePath);
 
-            $fileInfos = $this->findTwigFileInfos($absoluteTwigTemplatePath);
+            $files = $this->findTwigFiles($absoluteTwigTemplatePath);
```

Comment:
this file contains the relevant changes for this PR

Reply:
It's hard to see the changes. Please restore the original CS or re-run ECS.

Reply:
I restored all files, since I don't know ECS well enough to find the rules which you think about